SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1

```diff
@@ -26,17 +53,17 @@ function Invoke-lpass {
     
     if ($lpassCommand -ne '' -and ((& "$lpassCommand" $lpassPath --version ) -like 'LastPass CLI*') ) {
         if ($InputObject) {
-            $InputObject | & "$lpassCommand" $lpassPath @Arguments
+            return $InputObject | & "$lpassCommand" $lpassPath @Arguments 
         }
         else {
```

Comment:
Indeed, there was no technical reason. I won't usually put an else if I return but since both side were one liner and already part of a if in the parent level, it just felt easier to the eye to me. I removed them.